gammapy/image/maps.py

```diff
@@ -387,6 +387,48 @@ def cutout(self, position, size):
         skymap = SkyMap(data=cutout.data, wcs=cutout.wcs, unit=self.unit)
         return skymap
 
+    def pad(self, factor, mode, **kwargs):
```

Comment:
I think for a general public padding method this `factor` argument might be too special. It is hard to understand from the docs how it is actually used. Looking at the code I presume it something like: 
```
Pad image to the nearest larger shape, that is divisible by the given factor in both axis.
``` 
For a general purpose padding function if find a `pad_width` argument much more intuitive.